sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFluxBase.java

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.rest;
+
+import org.reactivestreams.Publisher;
+import reactor.core.CoreSubscriber;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+/**
+ * This class is a flux that can operate on a {@link PagedResponse} and
+ * also provides the ability to operate on individual items. When processing the response by page,
+ * each response will contain the items in the page as well as the request details like
+ * status code and headers.
+ *
+ * <p>To process one item at a time, simply subscribe to this flux as shown below </p>
+ * <p><strong>Code sample</strong></p>
+ * {@codesnippet com.azure.core.http.rest.pagedflux.items}
```

Comment:
Update codesnippet name to use `pagedfluxbase` instead.

Reply:
This is resolved.